app/models/discourse_voting/category_setting.rb

```diff
@@ -8,6 +8,8 @@ class CategorySetting < ActiveRecord::Base
 
     before_create :unarchive_votes
     before_destroy :archive_votes
+    before_destroy :log_category_voting_disabled
```

Comment:
Should this be `after_destroy` instead? I don't think it currently matters, but in theory a different `before_destroy` callback could cancel the removal, so the log entry would be invalid 🤷

Reply:
Great point! I've made the change.